src/main/drivers/sdmmc_sdio.h

```diff
@@ -216,22 +216,22 @@ typedef struct
 
 /* Prototype(s) -----------------------------------------------------------------------------------------------------*/
 
-extern           SD_CardInfo_t               SD_CardInfo;
-extern           SD_CardType_t               SD_CardType;
-
-void             SD_Initialize_LL            (DMA_Stream_TypeDef *dma);
-bool             SD_Init                     (void);
-bool             SD_IsDetected				(void);
-bool             SD_GetState                 (void);
-SD_Error_t       SD_GetCardInfo              (void);
-
-SD_Error_t       SD_ReadBlocks_DMA           (uint64_t ReadAddress, uint32_t *buffer, uint32_t BlockSize, uint32_t NumberOfBlocks);
-SD_Error_t       SD_CheckRead                (void);
-SD_Error_t       SD_WriteBlocks_DMA          (uint64_t WriteAddress, uint32_t *buffer, uint32_t BlockSize, uint32_t NumberOfBlocks);
-SD_Error_t       SD_CheckWrite               (void);
-
-SD_Error_t       SD_Erase                    (uint64_t StartAddress, uint64_t EndAddress);
-SD_Error_t       SD_GetCardStatus            (SD_CardStatus_t* pCardStatus);
+extern		 SD_CardInfo_t		     SD_CardInfo;
```

Comment:
Please check the indentation settings in your editor - the standard for Betaflight is to use 4 spaces, you have replaced a whole bunch of these with tabs.
Also, aligned indentation is generally discouraged, as this results in a lot of unneeded noise in reviews when indentation needs to be rearranged.